src/ctia/stores/es/crud.clj

```diff
@@ -190,6 +191,51 @@
    {:all-of {s/Any s/Any}
     :one-of {s/Any s/Any}}))
 
+(def sort-fields-mapping
+  "Mapping table for all fields which needs to be renamed
+   for the sorting. Instead of using fielddata we can have
+   a text field for full text searches, and an unanalysed keyword
+   field with doc_values enabled for sorting"
+  {"title" "title.whole"})
```

Comment:
This assumes that any type with a `title` field will be indexed with multifield, and at least one named `whole` like [this](https://github.com/threatgrid/ctia/blob/master/src/ctia/stores/es/mapping.clj#L64):
```
(def describable-entity-mapping
  {:title (assoc all_text
                 :fields {:whole all_token})
   :short_description all_text
   :description all_text})
```
Are all these entities the only ones that should refer it?
```
⋊> ~/w/ctia on es-migration-2556 ⨯ grep -r "describable-entity-mapping" src/*                                 14:19:52
src/ctia/entity/investigation.clj:     em/describable-entity-mapping
src/ctia/entity/sighting/es_store.clj:     em/describable-entity-mapping
src/ctia/entity/actor.clj:     em/describable-entity-mapping
src/ctia/entity/judgement/es_store.clj:     em/describable-entity-mapping
src/ctia/entity/relationship.clj:     em/describable-entity-mapping
src/ctia/entity/incident.clj:     em/describable-entity-mapping
src/ctia/entity/weakness/mapping.clj:     em/describable-entity-mapping
src/ctia/entity/data_table.clj:     em/describable-entity-mapping
src/ctia/entity/campaign.clj:     em/describable-entity-mapping
src/ctia/entity/casebook.clj:     em/describable-entity-mapping
src/ctia/entity/indicator.clj:     em/describable-entity-mapping
src/ctia/entity/vulnerability/mapping.clj:     em/describable-entity-mapping
src/ctia/entity/coa.clj:     em/describable-entity-mapping
src/ctia/stores/es/mapping.clj:(def describable-entity-mapping
```
My main concern is that it's hidden here.